src/oci/runtime/mod.rs

```diff
@@ -0,0 +1,77 @@
+//! OCI runtime implementations.
+
+use crate::oci::spec::runtime::LinuxResources;
+use anyhow::Result;
+use async_trait::async_trait;
+use strum::{AsRefStr, Display, EnumString, IntoStaticStr};
+use tokio::{process::Command, signal::unix::SignalKind};
+
+pub mod cli;
+
+#[async_trait]
+/// Runtime is the trait for implementing possible interactions with an OCI compatible container.
+pub trait Runtime {
```

Comment:
I think that this trait is more appropriately named as `Container` :)

Reply:
Yep, renamed it. :+1: